cron/stopStaleWorkspaces.sql

```diff
@@ -0,0 +1,11 @@
+-- BLOCK select_stale_workspaces
+SELECT
+    *
+FROM
+    workspaces AS w
+WHERE
+    w.state = 'running'::enum_workspace_state
+    AND (
+        w.launched_at < now() - ($launched_timeout_sec || ' seconds')::interval
+        OR w.heartbeat_at < now() - ($heartbeat_timeout_sec || ' seconds')::interval
```

Comment:
Avoiding string concatenation:

```suggestion
        w.launched_at < now() - make_interval(seconds => $launched_timeout_sec)
        OR w.heartbeat_at < now() - make_interval(seconds => $heartbeat_timeout_sec)
```